src/current/cockroachcloud/production-checklist.md

```diff
@@ -6,35 +6,78 @@ cloud: true
 docs_area: deploy
 ---
 
-This page provides important recommendations for CockroachDB {{ site.data.products.cloud }} production deployments.
+Before deploying CockroachDB {{ site.data.products.cloud }} in production, it is important to understand the Shared Responsibility Model that delineates the responsibilities of Cockroach Labs and the customer in managing CockroachDB {{ site.data.products.cloud }} clusters. 
 
-## Follow the SQL Best Practices
+Under the Shared Responsibility Model, Cockroach Labs is responsible for the following tasks: 
 
-To ensure optimal SQL performance for your CockroachDB {{ site.data.products.cloud }} cluster, follow the best practices described in the [SQL Performance Best Practices](https://www.cockroachlabs.com/docs/{{site.current_cloud_version}}/performance-best-practices-overview) guide.
+- Cluster and cloud service availability and reliability
+- Maintenance and security of hardware and operating systems
+- Database upgrades and security patches 
+- Daily and hourly cluster backups 
 
-## Use a pool of persistent connections
+The customer is responsible for the following tasks:
 
-Creating the appropriate size pool of connections is critical to gaining maximum performance in an application. Too few connections in the pool will result in high latency as each operation waits for a connection to open up. But adding too many connections to the pool can also result in high latency as each connection thread is being run in parallel by the system. The time it takes for many threads to complete in parallel is typically higher than the time it takes a smaller number of threads to run sequentially.
+- Sizing the workload correctly
```

Comment:
Is it sizing the workload or it sizing the cluster to fit the workload?

Reply:
Estimating the workload volumetrics as accurately as possible?